packages/aws_common/CHANGELOG.md

```diff
@@ -1,3 +1,14 @@
+## 0.4.0
+
+### Features
+- feat(aft): Version bump command ([#2068](https://github.com/aws-amplify/amplify-flutter/pull/2068))
+
+### Breaking Changes
+- fix(auth)!: Fetch Auth Session offline behavior ([#2585](https://github.com/aws-amplify/amplify-flutter/pull/2585))
```

Comment:
This is where isolating commits can be powerful. If the changes to aws_common were made in a separate commit, the version would not have needed a major bump. 

No worries at all since the version of the package doesn’t matter right now. Just a good thing to keep in mind in the future.

Reply:
As it is though, this should be removed from here.

Reply:
Yeah, I realized that after seeing this. To make sure we are on the same page, I will keep the version bump, but update the change log to not include this. Let me know if you think I should undo the version.

Reply:
Yeah just the change log entry